src/app/shared/services/shared.service.ts

```diff
@@ -182,6 +182,16 @@ export class SharedService {
       return this.rqManager.getv2(`orden-pago/${id}`, null, null, null, null, null, 0);
     }
 
+     /**
+   * Gets OrdenesPago by id
+   * @param id id de la orden de pago
+   * @returns  Documento
+   */
+      public getOrdenesPagoByDoc(doc: string) {
+        this.rqManager.setPath('CENTRAL_CUENTAS_CRUD_SERVICE');
+        return this.rqManager.getv2(`orden-pago/?query=DocumentoBeneficiario:${doc},Estado:Firmado`, null, null, null, null, null, 0);
```

Comment:
Ajustar el `rqManager.getv2` para que los parametros no especificados tomen los `null` y el `0`, prácticamente en el mismo orden que aparecen acá
De tal manera que acá se pueda hacer lo siguiente y no pase nada
```suggestion
        return this.rqManager.getv2(`orden-pago/?query=DocumentoBeneficiario:${doc},Estado:Firmado`);
```